org.coolero.Coolero.yaml

```diff
@@ -0,0 +1,109 @@
+app-id: org.coolero.Coolero
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: coolero.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Flatpak
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - '*.la'
+  - '*.a'
+modules:
+  - shared-modules/libusb/libusb.json
```

Comment:
you need to add the git submodule as well.

Reply:
:+1:  wasn't sure if I should add them here or not. I also added the flatpak-builder-tools submodule as I'll be using that later when updating dependencies.